packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.js

```diff
@@ -125,6 +125,10 @@ export default class QuanticResultQuickview extends LightningElement {
           },
         }
       );
+      this.addEventListener(
```

Comment:
This should not be included in the  `if (this.isResultAction)` condition block.

Reply:
I must have made a mis-type here, it was meant to be under the condition